components/Homepage/MessageOfTheDay.vue

```diff
@@ -1,10 +1,10 @@
 <template functional>
   <v-row class="justify-center">
     <div class="d-flex align-center">
-      <div class="title text-center mx-3 d-flex flex-column">
-        <span class="font-weight-light white--text">Open Platform Livescreaming</span>
-        <div>
-          <span class="overline font-weight-black grey--text rgb-text">viewer discretion is advised</span>
+      <div class="title text-center mx-3 my-3 d-flex flex-column">
+        <span class="font-weight-bold white--text display-1">Open Platform Livescreaming</span>
+        <div style="margin-top: -10px;">
+          <span class="overline font-weight-black rgb-text" style="font-size: 15pt">viewer discretion is advised</span>
```

Comment:
`font-size: 15pt`

oh god this hurts. Should be px / rem if possible. I don't think pt is used anywhere for sizing

Reply:
removed